packages/sdk-websigner/index.ts

```diff
@@ -0,0 +1,77 @@
+import * as crypto from 'crypto'
+import { CredentialSigner, KeyAssertion } from '@dfns/sdk'
+import { toBase64Url } from '@dfns/sdk/utils'
+
+export class WebKeySigner implements CredentialSigner<KeyAssertion> {
```

Comment:
nit: is the name "Web" clear that it's meant for "client-side" (browser-side). Maybe `BrowserKeySigner` + package `sdk-browser-keysigner` 🤷

Reply:
Updated